e2e/tests/notebook/notebook-create-python.spec.ts

```diff
@@ -76,22 +76,15 @@ describe('Create python kernel notebook', () => {
 
     expect(
       await notebook.runCodeCell(4, {
-        codeFile: 'resources/python-code/nbstripoutput-filter.py',
-        markdownWorkaround: true
-      })
-    ).toMatch(/success$/);
```

Comment:
Running `nbstripoutput-filter.py` is covered in `notebook-upload-file.spec`. This is going to make this test run much faster.